Sources/Contentful/Decodable.swift

```diff
@@ -34,19 +34,19 @@ public extension Decoder {
 
     /// The localization context of the connected Contentful space necessary to properly serialize
     /// entries and assets to Swift models from Contentful API responses.
-    public var localizationContext: LocalizationContext {
+    var localizationContext: LocalizationContext {
         return userInfo[.localizationContextKey] as! LocalizationContext
     }
 
     /// Helper method to extract the sys property of a Contentful resource.
-    public func sys() throws -> Sys {
+    func sys() throws -> Sys {
         let container = try self.container(keyedBy: LocalizableResource.CodingKeys.self)
         let sys = try container.decode(Sys.self, forKey: .sys)
         return sys
     }
 
     /// Extract the nested JSON container for the "fields" dictionary present in Entry and Asset resources.
-    public func contentfulFieldsContainer<NestedKey>(keyedBy keyType: NestedKey.Type) throws -> KeyedDecodingContainer<NestedKey> {
```

Comment:
I might be wrong, according to the Swift 5 docs the default accessor `internal` does this mean that all these func will be internal instead of public ?

Reply:
This is a totally valid concern! In Swift 4.2 there was a change regarding redundant visibility modifiers: https://bugs.swift.org/browse/SR-8453
So, the compiler started to warn when a `var` or `func` in an extension has the same visibility modifier as the extension itself, due to redundancy. These changes fix the warning. Visibility remains at `public`, which is inherited from the extension.